onnxruntime/core/session/inference_session.cc

```diff
@@ -995,6 +995,10 @@ const SessionOptions& InferenceSession::GetSessionOptions() const {
   return session_options_;
 }
 
+const DataTransferManager& InferenceSession::GetDataTransferManager() const {
+  return data_transfer_mgr_;
```

Comment:
Need DataTransferManager in pybind to do the device-host copy if the tensor is associated with memory in CUDA but the user requests a Numpy object